scripts/ansible

```diff
@@ -110,8 +112,8 @@ echo "deb http://ppa.launchpad.net/ansible/ansible/ubuntu $CODENAME main" \
 #echo "deb http://ppa.launchpad.net/ansible/ansible/ubuntu disco main" \
     #    > /etc/apt/sources.list.d/iiab-ansible.list
 
-echo -e '\nIF YOU FACE ERROR "signatures couldn'"'"'t be verified because the public key is not available" THEN REPEATEDLY RE-RUN "sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367"\n'
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367
+#echo -e '\nIF YOU FACE ERROR "signatures couldn'"'"'t be verified because the public key is not available" THEN REPEATEDLY RE-RUN "sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367"\n'
+#apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367
 echo -e 'PPA source "deb http://ppa.launchpad.net/ansible/ansible/ubuntu '$CODENAME' main"'
```

Comment:
Eventually we should make this Line 117...correspond to Lines 100-101 that it is documenting.

Reply:
Think the change should stay the way it is to denote a change in method of **using** the gpg key with signed-by= as opposed to the earlier method of trying to **install** the key. Keeps the commented code in one place